model/registry/log.yaml

```diff
@@ -70,3 +70,14 @@ groups:
           The id MAY be an [Universally Unique Lexicographically Sortable Identifier (ULID)](https://github.com/ulid/spec),
           but other identifiers (e.g. UUID) may be used as needed.
         examples: ["01ARZ3NDEKTSV4RRFFQ69G5FAV"]
+      - id: original
+        type: string
+        stability: experimental
+        brief: >
+          The complete orignal Log Record.
+        note: >
+          This value SHOULD be provided if the Log Record was originally transmitted as a string or equivalent data type AND
+          the Body field of the Log Record does not contain the same value. (e.g. a syslog or log record read from a file.)
```

Comment:
backends impose limits on attribute value length (smaller than on log body)  and it might be truncated. E.g. Az Monitor [limits](https://learn.microsoft.com/en-us/azure/azure-monitor/app/data-model-complete#custom-properties-4) attribute value to 8KB, event payload to 32KB, New Relic has other [limits](https://docs.newrelic.com/docs/data-apis/manage-data/view-system-limits/).

Given that log bodies could be huge, I could make sense to truncate them on the collection side or at least mention that truncation might happen on the backend.

Reply:
I'm not sure I agree with adding any language about truncation here. Any limit we would try to establish would be arbitrary in my opinion. I suppose we could mention backends but it seems like a given that backends will consume data according to their own limitations.